modules/support-network-trace-methods.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="support-network-trace-methods_{context}"]
+= Network trace methods
+
+Collecting network traces, in the form of packet capture records, can assist Red Hat Support with troubleshooting network issues.
+
+{product-title} supports two ways to perform a network trace.
+Review the following table to choose the method that meets your needs.
+
+.Supported methods to collect a network trace
+[cols="1,4a",options="header"]
+|===
+
+|Method
+|Benefits and capabilities
+
+|Collecting a host network trace
+|This method performs a packet capture for a duration that you specify on one or more nodes at the same time.
+The packet capture files are transferred from nodes to the client machine when the specified duration is met.
+
+If some activity triggers the network communication issue, such as accessing a web application, you can start the packet capture and then perform the action that triggers the issue.
```

Comment:
Since we're talking about benefits and capabilities, this might be stronger if we reframe the idea:

You can figure out why a specific action triggers a network communication issues. Start the packet capture, perform the action, and use use logs to diagnose the issue.